utils/markdown/renderer.tsx

```diff
@@ -207,8 +207,10 @@ export default class Renderer extends marked.Renderer {
 
         output += `" href="${outHref}" rel="noreferrer"`;
 
+        const pluginURL = `${this.formattingOptions.siteURL}/plugins`;
```

Comment:
To double check, do we normalize `siteURL` to remove trailing slashes that might be configured?

Reply:
from what I see, we get the siteurl from https://github.com/mattermost/mattermost-webapp/blob/b99ee5511507678edc1e3adb7d813a9f5ae139fd/utils/url.tsx#L36-L37

Reply:
And if it's coming from the server instead of the window, that also trims it
https://github.com/mattermost/mattermost-server/blob/master/config/client.go#L18